### PR TITLE
Add ROS service/client widget

### PR DESCRIPTION
**Pull request by ihuicatl**
The function generates a widget similar to the publisher/subscriber widget.

### CODE REVIEW

File: jupyros/ros_widgets.py [hunk 1]
```diff
@@ -287,4 +287,41 @@ def step_clk(arg):
...
+    widget_list = []
+    widget_dict = {}
+
+    add_widgets(srv_type._request_class(), widget_dict, widget_list)
+    call_btn = widgets.Button(description="Call Service")
+
+    def call_srv(arg):
+        msg_to_send = srv_type._request_class()
+        widget_dict_to_msg(msg_to_send, widget_dict)
+        rospy.wait_for_service(srv_name)
```
Comment (wolfv) [inline jupyros/ros_widgets.py:314]: we could move the wait_for_service call outside the callback, and we could also give a timeout (it's an optional parameter). 

What do you think?

File: notebooks/ROS Service.ipynb [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+       "VBox(children=(HBox(children=(Label(value='x', layout=Layout(width='100px')), FloatText(value=0.0))), HBox(chi…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
```
Comment (wolfv) [inline notebooks/ROS Service.ipynb:142]: You may want to clean this up :)

**Comment by wolfv** [comment]
Looks great to me!

File: jupyros/ros_widgets.py (hunk 1 above)
Comment (ihuicatl) [reply jupyros/ros_widgets.py:314]: Is it necessary to wait for the service? We could also simply check if the service is there and throw an error if it's not, would that be better?

**Pull request merged**